dtfs-central-api/src/services/portal/facility-amendment.service.ts

```diff
@@ -52,4 +58,33 @@ export class PortalFacilityAmendmentService {
 
     return amendmentToInsert;
   }
+
+  /**
+   * Updates a portal facility amendment with the provided details.
+   *
+   * @param params
+   * @param params.amendmentId - The amendment id
+   * @param params.facilityId - The facility id
+   * @param params.update - The update payload for the amendment.
+   * @param params.auditDetails - The audit details for the update operation.
+   * @returns A promise that resolves when the update operation is complete.
+   */
+  public static async updatePortalFacilityAmendment({
+    amendmentId,
+    facilityId,
+    update,
+    auditDetails,
+  }: {
+    amendmentId: string;
+    facilityId: string;
+    update: PortalFacilityAmendmentUserValues;
+    auditDetails: PortalAuditDetails;
+  }): Promise<void> {
+    const amendmentUpdate: Partial<PortalFacilityAmendment> = {
+      ...update,
+      updatedAt: getUnixTime(new Date()),
```

Comment:
Which helper method? Annoyingly the existing tfm amendments use unix epoch in seconds (because thats what `getUnixTime` returns) I want to keep these compatible so i've used the same here